### PR TITLE
Add Search for services component to `/service_search_widget` route for Drupal widget

### PR DESCRIPTION
**Pull request by SevaDer14**
PT: https://www.pivotaltracker.com/story/show/179400865
### Video

https://user-images.githubusercontent.com/79849155/131302117-cc86da4f-ce3d-4493-8071-05417fbaa0c5.mp4

### CODE REVIEW

File: src/Routes.jsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ import NewsView from "./views/NewsView";
 import ArticleView from "./views/ArticleView";
 import InformationView from "./views/InformationView";
 import ErrorView from "./views/ErrorView";
+import DrupalSearchWidget from "./views/DrupalSearchWidget";
```
Comment (tochman) [inline src/Routes.jsx:19]: I would skip the word `Drupal` in the name. Let's keep it generic

File: src/views/DrupalSearchWidget.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import ServiceSearch from "../components/ServiceSearch";
+import ServiceListing from "../components/ServiceListing";
+import { Container } from "@material-ui/core";
+
+const DrupalSearchWidget = () => {
+  return (
+    <Container maxWidth="md">
+    <ServiceSearch />
+    <ServiceListing />
+  </Container>
```
Comment (tochman) [inline src/views/DrupalSearchWidget.jsx:11]: Code formatting is off here.

**Review by tochman — CHANGES_REQUESTED** [review]

File: src/Routes.jsx (hunk 1 above)
Comment (SevaDer14) [reply src/Routes.jsx]: renamed

File: src/views/DrupalSearchWidget.jsx (hunk 2 above)
Comment (SevaDer14) [reply src/views/DrupalSearchWidget.jsx]: fixed

**Review by tochman — APPROVED** [review]

**Pull request merged**